pkg/virt-operator/install-strategy/create.go

```diff
@@ -2110,7 +2110,8 @@ func createOrUpdatePrometheusRules(kv *v1.KubeVirt,
 }
 
 // deprecated, keep it for backwards compatibility
-func addOrRemoveSSC(targetStrategy *InstallStrategy,
+func addOrRemoveSSC(targetNamespace string,
```

Comment:
Your logic is great. Based on the context of what has changed regarding SCC's I'm going to suggest a slightly different approach.

This Add/Remove SCC user logic was the only reason we needed the prevStrategy that I'm aware of. I don't ever anticipate us wanting this logic to alter existing SCCs again, so I think we should use this as an opportunity to get rid of it entirely.

The change I'm proposing would then be to
- remove usage of prevStrategy from both create.go and kubevirt.go
- remove the addOrRemoveSCC function
- add a new function that simply unconditionally removes our SAs from the privileged SCC if they are set (regardless of what is in the install strategy)